packages/web/src/pages/schedule.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Event, eventSchema, EventsList } from '../components/EventsList';
+import { PageContainer } from '../components/layout/PageContainer';
+
+const fetchEvents = async () => {
+  const res = await fetch('/api/event');
+
+  if (res.ok) {
+    const data: Event[] = await res.json();
+    const events: Event[] = (data).map((event) => eventSchema.parse(event));
+    
+    return events;
+  }
```

Comment:
With axios the result should be something like this:
```suggestion
  const { start, end, ...rest } = res;
  return data.map((serializedEvent) => {...rest, start: dayjs(start), end: dayjs(end) });
```

End result is an array of events that have dayjs objects so we won't need to use `Date` functionality anywhere